pkgs/by-name/dp/dprint/plugins/default.nix

```diff
@@ -32,12 +32,16 @@ let
         inherit initConfig updateUrl;
       };
       nativeBuildInputs = [ dprint ];
-      postFetch = ''
-        export DPRINT_CACHE_DIR="$(mktemp -d)"
-        cd "$(mktemp -d)"
-        cp '${formattedFile}' '${builtins.baseNameOf formattedFile}'
-        dprint check --plugins "$downloadedFile"
-      '';
+      postFetch =
+        if testFile != null then
+          ''
+            export DPRINT_CACHE_DIR="$(mktemp -d)"
+            cd "$(mktemp -d)"
+            cp '${testFile}' '${builtins.baseNameOf testFile}'
+            dprint check --plugins "$downloadedFile"
+          ''
+        else
+          "";
```

Comment:
Can we keep non nullable? I think there is no reason to skip tests

Reply:
If I am using mkDprintPlugin outside nixpkgs I don't want to write to a mandatory test.